crates/stark_hash/src/hash.rs

```diff
@@ -114,6 +114,13 @@ impl StarkHash {
         Ok(Self(bytes))
     }
 
+    /// Creates a [StarkHash] from a u64.
+    ///
+    /// Uses the big-endian representation, extended with zeroes.
+    pub fn from_u64(value: u64) -> Self {
```

Comment:
You could `impl std::convert::From<u64>`

Reply:
Thanks for the suggestion, done.